views/product-update.ejs

```diff
@@ -0,0 +1,53 @@
+<%- include('partials/header'); %>
+    <form action="<%= product.id %>/update" method="post" enctype="multipart/form-data">
+        <div class="mb-3">
+            <label for="name" class="form-label">Name</label>
+            <input type="text" class="form-control" name="name" value="<%= product.name %>">
+        </div>
+
+        <div class="mb-3">
+            <label for="description" class="form-label">Description</label>
+            <textarea name="description" class="form-control" id="" cols="30" rows="10"><%= product.description %></textarea>
+        </div>
+
+        <div class="mb-3">
+            <label for="price" class="form-label">Price</label>
+            <input type="number" class="form-control" name="price" value="<%= product.price %>">
+        </div>
+
+        <div class="mb-3">
+            <label for="note" class="form-label">Note</label>
+            <textarea name="note" class="form-control" id="" cols="30" rows="10"><%= product.note %></textarea>
+        </div>
+
+        <div class="mb-3">
+            <label for="category" class="form-label">Category</label>
+            <select name="category" class="form-control" id="category">
+                
```

Comment:
add a default option

```
<option>Not Assigned</option>
```
at the beginning, so that when people load this page, and the product does not have a value for the `product.categry` it will default to `Not Assigned` with an empty value.